virtualenv/builders/legacy.py

```diff
@@ -114,8 +115,8 @@ def _get_base_python_info(self):
         # Get information from the base python that we need in order to create
         # a legacy virtual environment.
         return json.loads(
-            subprocess.check_output([
-                self.python,
+            check_output([
+                self._get_base_python_bin(),
```

Comment:
I suppose we could special case this later on for Windows to avoid the secondary subprocess, but as it is now it's badly needed cause there's no reliable way to figure out where the bin really is without running it.

In my tests there wasn't noticeable slowdown.